resources/styleheets/main.css

```diff
@@ -17,6 +21,225 @@ h1, h2, h3, h4, h5, h6 {
   margin: 0;
 }
 
+body.login {
+  /* background: linear-gradient(to right, $background1, $background2); */
+  font-family: monospace;
+  background-image: url(resources/img/login_fondo.svg);
+  background-size: cover;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.contenido {
+  margin: 2.5rem;
+  height: 87vh;
+  width: 78%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 1rem;
+  color: white;
+  border: 0.05rem solid rgb(129, 129, 129);
+  box-shadow: rgba(0, 0, 0, 0.56) 0px 22px 70px 4px;
+  background-color: #080808;
+}
+
+.present {
+  height: 18rem;
+  order: 1;
+}
+
+.left {
+  flex: 0.7;
+  padding: 2rem;
+  display: grid;
+  align-items: center;
+  justify-content: center;
+}
+
+.right {
+  flex: 0.5;
+  border-radius: 2rem;
+  display: grid;
+  align-items: center;
+  justify-content: center;
+}
+.right .inicio {
+  font-size: 2em;
+  display: grid;
+  align-items: center;
+  justify-content: center;
+}
+.right .logo {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 0.9em;
+  color: white;
+  order: 3;
+}
+
+img {
+  height: 4.2rem;
+}
+
+.new {
+  color: #93F9B9;
+  margin-left: 0.5rem;
+  cursor: pointer;
+  text-decoration: none;
+}
+.new:hover {
+  color: white;
+}
```

Comment:
SASS syntax can be help to rewrite multiple times the same class
```
.class {
  &:selector {
  }
}
```
the & copy and paste the .class in the compiled CSS resulting the same you do above